src/main/java/seedu/address/logic/commands/meeting/EditCommand.java

```diff
@@ -0,0 +1,215 @@
+package seedu.address.logic.commands.meeting;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CONTACT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATETIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DURATION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MESSAGE;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Message;
+import seedu.address.model.Model;
+import seedu.address.model.meeting.Meeting;
+import seedu.address.model.person.Person;
+
+/**
+ * Edits the details of an existing meeting in the address book.
+ */
+public class EditCommand extends Command {
+
+    public static final String COMMAND_WORD = "meeting edit";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the meeting identified "
+            + "by the index number used in the displayed meeting list. "
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_CONTACT + "CONTACT_INDEX] "
+            + "[" + PREFIX_MESSAGE + "MESSAGE] "
+            + "[" + PREFIX_DATETIME + "DATETIME] "
+            + "[" + PREFIX_DURATION + "DURATION] "
+            + "Example: " + COMMAND_WORD + " 1 "
```

Comment:
Would be good to put the example in a separate line, currently it's cut off in the GUI!